example/blob_to_index.cpp

```diff
@@ -0,0 +1,134 @@
+#include <iostream>
+#include <fstream>
+#include <sys/stat.h>
+#include <sys/types.h>
+
+#include <eblob/eblob.hpp>
+
+
+static void print_usage(char *p)
+{
+	std::cerr << "Usage: " << p << " -i SRC -o DST\n\n"
+		"  This utility will create index from blob file\n\n"
+		"Options\n"
+		"  -i path             - input blob path\n"
+		"  -o path             - output index path\n"
+		"  -h                  - this help\n"
+		  << std::endl;
+	exit(-1);
+}
+
+off_t get_file_size(const std::string &filename)
+{
+	struct stat stat_buf;
+	int rc = stat(filename.c_str(), &stat_buf);
+	return rc == 0 ? stat_buf.st_size : -1;
+}
+
+inline bool check_record(const struct eblob_disk_control &dc, uint64_t offset, size_t blob_length)
+{
+	static const uint64_t hdr_size = sizeof(struct eblob_disk_control);
+	if (dc.disk_size < dc.data_size + hdr_size) {
+		std::cerr << "malformed entry: disk_size is less than data_size + hdr_size: "
```

Comment:
Can we also write the whole dc structure in case of error?
It helps understanding whether it is partial error or data is heavily corrupted